### PR TITLE
add support to send all request via forward proxy

### PR DESCRIPTION
**Pull request by ronkorland**

### CODE REVIEW

**Review by EldarAliiev — CHANGES_REQUESTED** [review]
Some tests are not passed, so we cannot merge without it.
The rest of code looks fine.
Another fix required: bump package version in package.json file, after that we will be able to merge and publish new version.

**Comment by ronkorland** [comment]
> Some tests are not passed, so we cannot merge without it.
> The rest of code looks fine.
> Another fix required: bump package version in package.json file, after that we will be able to merge and publish new version.

@EldarAliiev I fixed the tests + bump the version + looks like the tests in `http.service.js` doesn't really check anything because the code in sync and finish before the next or error call

**Review by EldarAliiev — APPROVED** [review]

**Comment by EldarAliiev** [comment]
Published in version 1.1.27

**Pull request merged**
